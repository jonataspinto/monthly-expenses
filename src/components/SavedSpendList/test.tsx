import { render } from '@testing-library/react'

import { SavedSpendList } from '.'

describe('<SavedSpendList />', () => {
  it('should render the heading', () => {
    const { container } = render(
      <SavedSpendList sheetRows={[]} savedSpendList={[]} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
