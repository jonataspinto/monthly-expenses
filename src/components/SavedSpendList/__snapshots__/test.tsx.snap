// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SavedSpendList /> should render the heading 1`] = `
.c0 {
  width: 100%;
  max-width: 900px;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 0.75rem;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 h2 {
  font-size: 2rem;
  font-weight: 400;
}

.c1 {
  display: grid;
  grid-template-columns: repeat(4,1fr);
  grid-template-rows: repeat(10,1fr);
  gap: 0.75rem;
  margin-top: 0.75rem;
  height: 320px;
  overflow-y: auto;
}

@media (max-width:768px) {
  .c1 {
    grid-template-columns: repeat(3,1fr);
    grid-template-rows: repeat(6,1fr);
  }
}

@media (max-width:425px) {
  .c1 {
    grid-template-columns: repeat(2,1fr);
  }
}

<div
  class="c0"
>
  <h2>
    Ultimos Lan√ßamentos
  </h2>
  <div
    class="c1"
  />
</div>
`
